QUICKSTART.md

```diff
@@ -0,0 +1,88 @@
+# Building the Repository
+
+To build the entire repository and run the application, follow these steps:
+
+1. The 2 main system dependencies to install are:
+
+   - [volta](https://docs.volta.sh/guide/getting-started)
+   - [docker](https://docs.docker.com/get-docker/)
+
+2. Clone the repo:
+
+   ```zsh
+   git clone https://github.com/cardstack/boxel.git
+   ```
+
+3. Install the package dependencies:
+
+   ```zsh
+   echo 'export VOLTA_FEATURE_PNPM=1"' >> ~/.zprofile && source ~/.zprofile
```

Comment:
this assumes you are using zsh. probably you can make more more generic instructions `$HOME/.profile` is much more universal